src/ophys_etl/modules/segmentation/qc_utils/roi_comparison_utils.py

```diff
@@ -60,12 +66,73 @@ def _validate_paths_v_names(
     return paths, names
 
 
+def get_roi_color_map(
+        roi_list: List[OphysROI]) -> Dict[int, Tuple[int, int, int]]:
+    """
+    Take a list of OphysROI and return a dict mapping ROI ID
+    to RGB color so that no ROIs that touch have the same color
+
+    Parametrs
+    ---------
+    roi_list: List[OphysROI]
+
+    Returns
+    -------
+    color_map: Dict[int, Tuple[int, int, int]]
+    """
+    roi_graph = networkx.Graph()
+    for roi in roi_list:
+        roi_graph.add_node(roi.roi_id)
+    for ii in range(len(roi_list)):
+        roi0 = roi_list[ii]
+        for jj in range(ii+1, len(roi_list)):
+            roi1 = roi_list[jj]
+
+            # value of 5 is so that singleton ROIs that
+            # are near each other do not get assigned
+            # the same color
+            abut = do_rois_abut(roi0, roi1, 5.0)
```

Comment:
How is a pixel value of `5.0` determined empirically? Are there edge cases where this value might not catch `abut`ing ROIs?

Reply:
It was just chosen out of a hat.

Note: the effect of this parameter is actually to trick `do_rois_abut` into thinking that ROIs that do not abut, but are separated by <= 5 pixels are neighbors. This is so that, if you have a dense cluster of ROIs that aren't actually touching, `networkx` won't blithely assign them all the same color (that actually happened before I set this value)

<img width="129" alt="Screen Shot 2021-07-28 at 12 57 47 PM" src="https://user-images.githubusercontent.com/1682854/127387546-20542e15-5261-49a1-a55f-da420e23cc62.png">

vs

<img width="166" alt="Screen Shot 2021-07-28 at 12 58 22 PM" src="https://user-images.githubusercontent.com/1682854/127387627-cf8a51ef-110d-41fa-8813-6adad4dec941.png">

Shall I just expose this parameter to the user with a default value of `5`?